examples/hubtype_demo/package.json

```diff
@@ -0,0 +1,99 @@
+{
+  "scripts": {
+    "build": "webpack --env.target=all --mode=production",
+    "build_development": "webpack --env.target=node --mode=development",
+    "build_with_tests": "tsc -b tests/tsconfig.json",
+    "lint": "node_modules/.bin/eslint --fix --quiet 'src/**/*.ts*' 'tests/**/*.ts*'",
```

Comment:
add --cache flag for faster performance

Reply:
do 
`npm i -D eslint_d`
and replace node_modules/.bin/eslint node_modules/.bin/eslint_d for even faster performance :-)
https://www.npmjs.com/package/eslint_d